### PR TITLE
CRS-516 Add external dns annotations for preprod - as advised by CP team

### PR DESCRIPTION
**Pull request by yasinmustafa**
As per this guide https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/migrate-to-live-ingress-annotation.html#add-external-dns-annotations-to-your-ingress-resource-in-live-1

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ generic-service:
 
   ingress:
     host: calculate-release-dates-api-preprod.hmpps.service.justice.gov.uk
+    annotations:
+      external-dns.alpha.kubernetes.io/set-identifier: calculate-release-dates-api-calculate-release-dates-api-preprod-green
+      external-dns.alpha.kubernetes.io/aws-weight: 100
```
Comment (timharrison-moj) [inline helm_deploy/values-preprod.yaml:12]: I think you can add these simply by specifying `contextColour: green` - the generic-service takes care of actually adding the annotations. Take a look at `create-and-vary-a-licence-api` as an example. Make sure to use the latest generic service version too.

Comment (yasinmustafa) [reply helm_deploy/values-preprod.yaml:10]: Thanks @timharrison-moj - have changed it

Comment (timharrison-moj) [reply helm_deploy/values-preprod.yaml:10]: Actually - sorry - you're still in live-1 cluster?  That means it should be contextColour = blue.

Comment (timharrison-moj) [reply helm_deploy/values-preprod.yaml:10]: No ingress for your API service in the live cluster :-( - I just checked.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**